streams/src/main/java/org/apache/kafka/streams/KeyQueryMetadata.java

```diff
@@ -50,32 +50,65 @@ public KeyQueryMetadata(final HostInfo activeHost, final Set<HostInfo> standbyHo
     }
 
     /**
-     * Get the Active streams instance for given key
```

Comment:
As the original PR is merged already, this seems to be redundant (similar below for all other code changes). -- This PR should only update the html file?

Reply:
Yeah , this should only update the html file. I raised the PR from the same branch. Err., I wonder, as these changes were already merged, should it even be showing these as changes here ? I'm saying about the previous changes which were already merged.

Reply:
If you see from the commit history in the timeline . My last commit is : https://github.com/apache/kafka/pull/9133/commits/14ab0529a492e9b1d361b1ab3081b8fa7189632f (this has only the html changes) and all the ones before that are already merged to trunk.

Reply:
Not sure. I am not an git guru... Can you maybe just rebase this PR against latest trunk (just to make sure we don't mess anything up?)